Dockerfile

```diff
@@ -56,8 +56,15 @@ EXPOSE 4567-4584 8080
 # install supervisor daemon & copy config file
 ADD bin/supervisord.conf /etc/supervisord.conf
 
+RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
```

Comment:
Is the `pip` installation required here? (pip should already be available from the base image).

Could this simply be:
```
RUN pip install awscli awscli-local --upgrade --user
```
?

Reply:
You are right. pip is available already. I've removed installation of pip.

Reply:
Thanks